lib/ansible/modules/cloud/azure/azure_rm_keyvault.py

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_keyvault
+version_added: "2.5"
+short_description: Manage Key Vault instance.
+description:
+    - Create, update and delete instance of Key Vault.
+
+options:
+    resource_group:
+        description:
+            - The name of the Resource Group to which the server belongs.
+        required: True
+    vault_name:
+        description:
+            - Name of the vault
+        required: True
+    location:
+        description:
+            - Resource location. If not set, location from the resource group will be used as default.
+    tenant_id:
```

Comment:
change this to `vault_tenant`. `tenant_id` is too close to the base option `tenant` used as a credential arg.